protocol-units/execution/opt-executor/src/executor/services.rs

```diff
@@ -14,15 +15,29 @@ impl Executor {
 	pub async fn run_service(&self) -> Result<(), anyhow::Error> {
 		info!("Starting maptos-opt-executor services at: {:?}", self.listen_url);
 
+		let size_limit = self.context.content_length_limit();
```

Comment:
This does not seem to be used for anything.

Reply:
Yeah, sorry, it should have been used the same way as in Aptos, but it involves making visibility changes that I didn't have time for. I can remove later.